src/Uno.UI.FluentTheme/Resources/PriorityDefault/FlipView_themeresources.xaml

```diff
@@ -317,6 +319,7 @@
                             ZoomMode="Disabled"
                             TabNavigation="{TemplateBinding TabNavigation}"
                             IsTabStop="False"
+							ForceChangeToCurrentView="True"
```

Comment:
I noticed that this is an internal property - in that case it shouldn't be set from the style, because then custom styles won't be able to work. If it's required by FlipView, you should just set it from code.

Reply:
Updating property from Code

Reply:
Completed